ts/features/bonus/siciliaVola/screens/voucherList/VoucherDetailsScreen.tsx

```diff
@@ -0,0 +1,202 @@
+import { H1 } from "../../../../../components/core/typography/H1";
+import { GlobalState } from "../../../../../store/reducers/types";
+import {
+  svGenerateVoucherBack,
+  svGenerateVoucherCompleted,
+  svGenerateVoucherGeneratedVoucher
+} from "../../store/actions/voucherGeneration";
+import FooterWithButtons from "../../../../../components/ui/FooterWithButtons";
+import { VoucherRequest } from "../../types/SvVoucherRequest";
+import I18n from "../../../../../i18n";
+import { svVoucherDetailGet } from "../../store/actions/voucherList";
+import { SvVoucherId } from "../../types/SvVoucher";
+import {
+  selectedVoucherCodeSelector,
+  selectedVoucherSelector
+} from "../../store/reducers/selectedVoucher";
+import { H3 } from "../../../../../components/core/typography/H3";
+import CopyButtonComponent from "../../../../../components/CopyButtonComponent";
+import { isLoading, isReady } from "../../../bpd/model/RemoteValue";
+import { H4 } from "../../../../../components/core/typography/H4";
+import { formatDateAsLocal } from "../../../../../utils/dates";
+import { useIOBottomSheetRaw } from "../../../../../utils/bottomSheet";
+import { IOColors } from "../../../../../components/core/variables/IOColors";
+import { LoadingErrorComponent } from "../../../bonusVacanze/components/loadingErrorScreen/LoadingErrorComponent";
+import VoucherDetailBottomSheet from "../../components/VoucherDetailBottomsheet";
+import { fromVoucherToDestinationLabels } from "../../utils";
+import { navigateToWorkunitGenericFailureScreen } from "../../../../../store/actions/navigation";
+
+type Props = ReturnType<typeof mapDispatchToProps> &
+  ReturnType<typeof mapStateToProps>;
+
+const styles = StyleSheet.create({
+  itemRow: {
+    flex: 1,
+    flexDirection: "row",
+    justifyContent: "space-between"
+  }
+});
+
+const VoucherDetailsScreen = (props: Props): React.ReactElement | null => {
+  useEffect(() => {
+    if (props.selectedVoucherCode !== undefined) {
+      props.getVoucherDetail(props.selectedVoucherCode);
+    }
+  }, []);
+  const { present, dismiss } = useIOBottomSheetRaw(650);
+
+  // The selectedVoucherCode can't be undefined in this screen
+  if (!isReady(props.selectedVoucher)) {
+    return fromNullable(props.selectedVoucherCode).fold(null, svc => (
+      <BaseScreenComponent
+        goBack={true}
+        contextualHelp={emptyContextualHelp}
+        headerTitle={I18n.t("bonus.sv.headerTitle")}
+      >
+        <LoadingErrorComponent
+          isLoading={isLoading(props.selectedVoucher)}
+          loadingCaption={I18n.t("global.remoteStates.loading")}
+          onRetry={() => props.getVoucherDetail(svc)}
+        />
+      </BaseScreenComponent>
+    ));
+  }
+
+  const selectedVoucher = props.selectedVoucher.value;
+  const openBottomSheet = () =>
+    present(
+      <VoucherDetailBottomSheet
+        barCode={selectedVoucher.barCode}
+        qrCode={selectedVoucher.qrCode}
+        onExit={() => dismiss()}
```

Comment:
```suggestion
        onExit={dismiss}
```

Reply:
Updated in [aff981c](https://github.com/pagopa/io-app/pull/3400/commits/aff981c96caa17dc974f5fabcbf0b37806ea5b78)